### ISSUE TITLE
"marketplace" app (idea for future)

### ISSUE THREAD

**Issue by ngvincent** [ma1co/PMCADemo#5]
Currently, we need tethered-usb to install (maybe thru wifi as well?)

We should look at how/whether a (marketplace) app can download&install other apps. Can the app bypass the RSA/DSA key and simply execute the default android install function?(RSA/DSA key is same for all users?)

The marketplace app can simply be a QR code reader. The marketplace webpage can display a QR code (pointing to correct ?RSA/DSA-encrypted? spk/apk file) and install

That way, user doesn't have to scroll through a gigantic list of apps on the camera's marketplace app

**Comment by ma1co** [comment]
Yes, this is possible. However, only .spk packages are supported by the installer. So we still have to encrypt our apks into spks (you can use PMCA-RE to do this).

I've implemented InstallerActivity as a proof of concept. It allows you to install apps from your SD card. It won't be much trouble to add the wifi download later.
